### PR TITLE
rtt updates/localization for Gele Plateau/Nefritas Cliff/Tenet Garden

### PR DESCRIPTION
**Pull request by terrapin47**

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -7549,10 +7549,10 @@ ETC_20150317_007549	Listen to the next plan from Commander Raimondas
 ETC_20150317_007550	Find the animal's dead body
 ETC_20150317_007551	Deliver the Divine Sphere to Paladin (1)
 ETC_20150317_007552	Deliver the Divine Sphere to Paladin (2)
-ETC_20150317_007553	Charge the Tree Guard Barrier with demon souls
+ETC_20150317_007553	Charge the barrier at Tree Guard Post with demon souls
```
Comment (ttgmichael) [inline ETC.tsv]: Barrier of Tree Guard Post?

File: ETC.tsv [hunk 2]
```diff
@@ -12747,14 +12747,14 @@ ETC_20150714_012748	Using Zubeck's Secret Moves has given you a status point!
...
 ETC_20150714_012750	Gesti is nearby{nl}Be careful not to get caught!
 ETC_20150714_012751	You have obtained a piece of the barrier.{nl}Insert it into the central pillar.
-ETC_20150714_012752	You can't destroy it due to the energy of the demon that is near the Panto Totem.
+ETC_20150714_012752	You can't destroy the Panto Totem due to the demonic energy surrounding it.
 ETC_20150714_012753	$The demon soul has been eradicated.
 ETC_20150714_012754	$The demon souls were severed by the barrier's holy power.{nl}Defeat the severed souls!
 ETC_20150714_012755	$The barrier powers can be only activated when there are weakened monsters nearby
 ETC_20150714_012756	$You obtained a piece of the destroyed barrier
-ETC_20150714_012757	$You charged the Tree Guard Barrier with a demon soul
+ETC_20150714_012757	$You charged the barrier at Tree Guard Post with a demon soul
```
Comment (ttgmichael) [inline ETC.tsv]: Barrier of Tree Guard Post?

File: ETC.tsv [hunk 3]
```diff
@@ -12747,14 +12747,14 @@ ETC_20150714_012748	Using Zubeck's Secret Moves has given you a status point!
...
 ETC_20150714_012758	$You need to defeat the demons near the Barrier to charge it.
-ETC_20150714_012759	$You've charged the Tree Guard Barrier a bit
+ETC_20150714_012759	$You've charged the barrier at Tree Guard Post a bit
```
Comment (ttgmichael) [inline ETC.tsv]: Barrier of Tree Guard Post?

File: QUEST_LV_0100.tsv [hunk 4]
```diff
@@ -919,28 +919,28 @@ QUEST_LV_0100_20150317_000914	{memo X}When you find the revelation, tell the Pal
...
+QUEST_LV_0100_20150317_000927	I'm sorry. It's all my fault. {nl}Gesti has realized that the Holy Relic is not the revelation, so she will head straight to the chapel. {nl}
 QUEST_LV_0100_20150317_000928	{memo X}But this old body of mine does not follow my will anymore. {nl} Now I can only trust my old friend Algis and you. {nl}
 QUEST_LV_0100_20150317_000929	{memo X}
 QUEST_LV_0100_20150317_000930	It is too much for me to handle alone without any help from the other brothers. {nl} I'd better use the Malda altar. {nl}
 QUEST_LV_0100_20150317_000931	{memo X}
 QUEST_LV_0100_20150317_000932	We still have a lot to do here.{nl}The Goddess told us to never give up.
 QUEST_LV_0100_20150317_000933	The world still needs more experience. {nl} We probably wouldn't have made it this far without the help of the Goddess.
-QUEST_LV_0100_20150317_000934	Activating the Tree Guard Barrier is taking longer than I thought. {nl} We have no time, let's charge it with demon souls instead.
+QUEST_LV_0100_20150317_000934	Activating the barrier at Tree Guard Post is taking longer than I thought. {nl} We have no time, let's charge it with demon souls instead.
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv]: Barrier of Tree Guard Post

File: QUEST_LV_0100.tsv [hunk 5]
```diff
@@ -919,28 +919,28 @@ QUEST_LV_0100_20150317_000914	{memo X}When you find the revelation, tell the Pal
...
 QUEST_LV_0100_20150317_000935	All souls are pure. {nl} So even if it's a demon's soul, it can still provide some divine power.
-QUEST_LV_0100_20150317_000936	So the Tree Guard Barrier had been charged with demon souls, huh. {nl} It still needs some final touches for completion, but I'll take care of it from here. {nl}Great job.
+QUEST_LV_0100_20150317_000936	So the barrier at Tree Guard Post has been charged with demon souls, huh. {nl} It still needs some final touches for completion, but I'll take care of it from here. {nl}Great job.
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv]: Barrier of Tree Guard Post

File: QUEST_LV_0100.tsv [hunk 6]
```diff
@@ -952,9 +952,9 @@ QUEST_LV_0100_20150317_000947	According to the elders, we are done with those Pa
...
 QUEST_LV_0100_20150317_000949	However, it's too dangerous to enter the basement with the Divine Sphere. {nl} I'll wait here so you go to the 1st floor through the basement and open the barrier. {nl}
 QUEST_LV_0100_20150317_000950	You have done what no other could easily do.{nl}The Goddess must be proud of you.
-QUEST_LV_0100_20150317_000951	I would say that my faith is like a first-class blade that can cut through these demons, but the reality isn't that simple.{nl}I hope you help us a bit.
+QUEST_LV_0100_20150317_000951	I would like to say that my faith is like a first-class blade that can cut through these demons, {nl}but in reality it isn't that simple. I hope you'll help us some.
 QUEST_LV_0100_20150317_000952	{memo X}
-QUEST_LV_0100_20150317_000953	Defeat the demons around the Tree Guard Barrier. {nl} The souls of those demons should be able to fill its divine powers a little. I'm counting on you!
+QUEST_LV_0100_20150317_000953	Defeat the demons around the barrier at Tree Guard Post. {nl} The souls of those demons should be able to fill its divine powers a little. I'm counting on you!
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv]: Barrier of Tree Guard Post

File: QUEST_LV_0100.tsv [hunk 7]
```diff
@@ -5151,18 +5151,18 @@ QUEST_LV_0100_20150317_005146	{memo X}
...
 QUEST_LV_0100_20150317_005149	$Collect the pieces of the destroyed barrier
-QUEST_LV_0100_20150317_005150	$Guard Allen asked you to collect the pieces of barrier the demons destroyed in Mazas Rest Place and take them to Kayetonas.
+QUEST_LV_0100_20150317_005150	$Guard Allen asked you to collect the pieces of barrier the demons destroyed at Mazas Rest Place and take them to Kayetonas.
 QUEST_LV_0100_20150317_005151	Talk to Follower Kayetonas
-QUEST_LV_0100_20150317_005152	$Collected the pieces of barrier. Give the pieces to Kayetonas.
-QUEST_LV_0100_20150317_005153	$Charge the Tree Guard Barrier
-QUEST_LV_0100_20150317_005154	$Help Allen defeat the demons around the Tree Guard Barrier and charge the barrier with demon souls. 
-QUEST_LV_0100_20150317_005155	$Tree Guard Barrier has been charged. Inform Kayetonas about it. 
+QUEST_LV_0100_20150317_005152	$Collected the pieces of the barrier. Give the pieces to Kayetonas.
+QUEST_LV_0100_20150317_005153	$Charge the barrier at Tree Guard Post
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv]: Barrier of Tree Guard Post

File: QUEST_LV_0100.tsv [hunk 8]
```diff
@@ -5151,18 +5151,18 @@ QUEST_LV_0100_20150317_005146	{memo X}
...
+QUEST_LV_0100_20150317_005154	$Help Guard Allen defeat the demons around the barrier at Tree Guard Post and charge the barrier with demon souls. 
+QUEST_LV_0100_20150317_005155	$The barrier at Tree Guard Post has been charged. Inform Kayetonas about it.
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv]: Barrier of Tree Guard Post

File: QUEST_LV_0100.tsv [hunk 9]
```diff
@@ -5151,18 +5151,18 @@ QUEST_LV_0100_20150317_005146	{memo X}
...
-QUEST_LV_0100_20150317_005150	$Guard Allen asked you to collect the pieces of barrier the demons destroyed in Mazas Rest Place and take them to Kayetonas.
+QUEST_LV_0100_20150317_005150	$Guard Allen asked you to collect the pieces of barrier the demons destroyed at Mazas Rest Place and take them to Kayetonas.
 QUEST_LV_0100_20150317_005151	Talk to Follower Kayetonas
-QUEST_LV_0100_20150317_005152	$Collected the pieces of barrier. Give the pieces to Kayetonas.
-QUEST_LV_0100_20150317_005153	$Charge the Tree Guard Barrier
-QUEST_LV_0100_20150317_005154	$Help Allen defeat the demons around the Tree Guard Barrier and charge the barrier with demon souls. 
-QUEST_LV_0100_20150317_005155	$Tree Guard Barrier has been charged. Inform Kayetonas about it. 
+QUEST_LV_0100_20150317_005152	$Collected the pieces of the barrier. Give the pieces to Kayetonas.
+QUEST_LV_0100_20150317_005153	$Charge the barrier at Tree Guard Post
+QUEST_LV_0100_20150317_005154	$Help Guard Allen defeat the demons around the barrier at Tree Guard Post and charge the barrier with demon souls.
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv]: Barrier of Tree Guard Post

**Comment by terrapin47** [comment]
@imcgames Can you give me the exact definition of 치유의 성물? And is it related to 성구 (Divine Sphere)?

Currently, this PR is standardizing it to "Holy Relic" ( and the others to "Divine Sphere" ), but it appears that they are the same thing in the story.  Are they just referred to differently, or am I wrong?

**Comment by Sunaries** [comment]
:+1: Looks good. But yeah, we need clarification from IMC on 치유의 성물 and 성구. 성구 is closely interpreted as Divine Sphere, but not sure about 치유의 성물.

**Comment by terrapin47** [comment]
Also the 'Divine Sphere' looks more like a carved horn or something.  Is this accurate?

**Comment by imcgames** [comment]
성구 is not necessarily a sphere (round). It's just a holy/sacred/divine 'thing' in general and yeah, it looks like a carved horn. 
As for 치유의 성물, it's something like a 'divine relic/object of healing' (definitely needs a better English name :laughing: ) that's buried in the ground used to lure Gesti.

**Comment by terrapin47** [comment]
Alright thanks, then this PR should be good to merge, and we can find/replace the standardized names when we figure em out.

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**